presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -167,9 +167,9 @@ public BaseJdbcClient(
     }
```

Comment:
Do you think it make sense to divide this commit into two? One for jdbc client and other for connection factory?

Reply:
jdbc changes are addressed here:
https://github.com/prestosql/presto/pull/6078
https://github.com/zaz968m/presto/pull/2/files